constants/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .gene_signature_definitions import GENE_SIGNATURE_DEFINITIONS
```

Comment:
is there a benefit of importing functions through these dictionaries rather than using simple import statements? I am curious